src/app/components/location-link/location-link.component.ts

```diff
@@ -77,12 +84,16 @@ export class LocationLinkComponent implements OnInit {
 
                 if (path && path.name && path.elements) {
                     this.displayText = this.getDisplayText(path);
-                    this.tooltip = this.getTooltip(path);
+                    this._path = path;
                 }
             }
         }
     }
 
+    ngOnDestroy() {
+        this.nodeLocation$.unsubscribe();
```

Comment:
this is not needed with using with async pipes as they know how to unsubscribe correctly